src/distilabel/pipeline/llm/vllm.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2023-present, Argilla, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Dict, Optional
+
+from vllm import LLM as _vLLM
+from vllm import SamplingParams
+
+from distilabel.pipeline.llm.base import LLM
+from distilabel.pipeline.step.task.typing import ChatType
+
+
+class vLLM(LLM):
+    model: str
+    model_kwargs: Optional[Dict[str, Any]] = {}
+    chat_format: Optional[str] = None
+
+    def load(self) -> None:
+        self._model = _vLLM(self.model, **self.model_kwargs)  # type: ignore
+        self._tokenizer = self._model.get_tokenizer()
+        self._values["model_name"] = self.model
```

Comment:
Poll time! What do you prefer here, to re-use the `_values["model_name"]` or to define here the `property` instead?

i.e. the current over the following? or the other way around?

```python
@property
def model_name(self) -> str:
    return self.model
```

Reply:
I personally prefer de property `model_name` because I find it more straightforward, but not a strong opinion against the `self._values`

Reply:
Cool, then feel free to update the rest of the `LLM`s so that we use the property instead 🤗